lib/practice_exercises.rb

```diff
@@ -1,13 +1,65 @@
 
-# Time Complexity: ?
-# Space Complexity: ?
-def is_palindrome(string)
-  raise NotImplementedError, "Not implemented yet"
+# Time Complexity: O(n)
+# Space Complexity: 0(1)
+
+#determine if string is a palindrome
+#input "string"
+#return true if palindrome
+#return false if not
+
+def is_palindrome(aString)
```

Comment:
👍 , although since you are duplicating the string with `.downcase` the space complexity is O(n)